homeassistant/components/onkyo/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow:
+    status: todo
+    comment: |
+      The data_descriptions are missing.
+  config-flow-test-coverage:
+    status: todo
+    comment: |
+      Coverage is 100%, but the tests need to be improved.
+  dependency-transparency: done
+  docs-actions: done
```

Comment:
The docs seem to put everything under the `options` field, while I think the actions can deserve their own chapter.

![image](https://github.com/user-attachments/assets/0979a1f2-b84b-4554-be73-00d9e656cc38)